tests/mail/tests.py

```diff
@@ -392,12 +413,49 @@ def test_attach_file(self):
 
     def test_attach_text_as_bytes(self):
         msg = EmailMessage('subject', 'body', 'from@example.com', ['to@example.com'])
-        file_path = os.path.join(os.path.dirname(upath(__file__)), 'attachments', 'file.txt')
-        with open(file_path, mode='rb') as fh:
-            msg.attach('file.txt', fh.read())
+        msg.attach('file.txt', b'file content')
+
+        # Check that the message would be sent at all.
         sent_num = msg.send()
         self.assertEqual(sent_num, 1)
 
+        filename, content, mimetype = self.get_decoded_attachments(msg)[0]
+
+        self.assertEqual(filename, 'file.txt')
+        self.assertEqual(content, b'file content')
+        self.assertEqual(mimetype, 'text/plain')
+
+    def test_attach_utf8_text_as_bytes(self):
+        """
+        Check whether non-ASCII characters encoded as valid UTF-8 get correctly
```

Comment:
State the expected behavior rather than "Checks that" or "Tests that" since all tests have that purpose.